### DISCUSSION TITLE
Auto refresh chatter list

### DISCUSSION THREAD

**Discussion by pajlada** [Chatterino/chatterino2#4641] [Ideas]
It was brought up in the Chatterino Discord that it might be useful to auto refresh the chatter list once it's open.

Open questions:
If we implement this, would anyone want to disable it?
How often can we reasonably refresh this?

**Comment by Mm2PL** [comment]
Having an option to take a snapshot of the users in chat might be useful for stuff like raids. If the chatter list were to refresh automatically I think it would be really nice if it would highlight users who join/leave maybe something vaguely similar to what htop does for processes.


The old tmi api was cached for 5 minutes and so was the webchat list at the time. I wouldn't expect it to refresh sooner. It might be useful to dynamically find when the list was cached so it could be refreshed just as it leaves the cache for optimal *requests/useful updates*.

**Comment by GoodCharles** [comment]
Thank you so much for opening this question for reviews !
I'd be extremely happy as a small streamer to use Chatterino as my main program for reading chat and the implementation of auto-refreshing viewers list would be an absolutely amazing feature to have !

I hope you wonderful people will make it a reality even if the refreshes occur 1 to 2 minutes or anything at all really..

Thank you so much all !

**Comment by 8thony** [comment]
Related #1190

**Reply by GoodCharles** [reply]
Oh my.. 2019?
Not sure if we have a chance at this being realized .. still hopeful though
Thank you for your addition
